extension/observer/cfgardenobserver/README.md

```diff
@@ -0,0 +1,53 @@
+# CF Garden Observer Extension
```

Comment:
Nit: We may want to expand Cloud Foundry here instead of just using the initials, to make it more clear what we're referring to.